package.json

```diff
@@ -82,5 +82,9 @@
     "react-native-windows": "^0.64.3",
     "react-native-macos": "^0.64.6",
     "simple-plist": "2.0.0-rc.0"
+  },
+  "dependencies": {
+    "gulp": "^4.0.2",
+    "gulp-rename": "^2.0.0"
```

Comment:
why this change?

Reply:
This was from one of the commands run to use the component generator

Reply:
I would also delete these from this change